cli/server_test.go

```diff
@@ -429,6 +429,39 @@ func TestServer(t *testing.T) {
 		cancelFunc()
 		<-serverErr
 	})
+	t.Run("GitHubOAuth", func(t *testing.T) {
```

Comment:
so this is an entirely new test for all GitHub OAuth?

Reply:
Yup! The OAuth callback code itself is tested quite extensively in the other handler, so this just ensures it enables properly, and does a correct callback URL for Enterprise.